pkgs/applications/version-management/gitea/default.nix

```diff
@@ -16,23 +16,44 @@
 , brotli
 , xorg
 , nixosTests
+, buildNpmPackage
 }:
 
-buildGoModule rec {
+let
+  frontend = buildNpmPackage {
+    inherit (gitea) src version;
+    pname = "gitea-frontend";
+    npmDepsHash = "sha256-gXBBiDIIS0aW6qK37HcF0AuJOliblinznRVXoo6DV1s=";
+    # use webpack directly instead of 'make frontend' as the packages are already installed
+    buildPhase = ''
+      BROWSERSLIST_IGNORE_OLD_DATA=true npx webpack
+    '';
+    installPhase = ''
+      mkdir -p $out
+      cp -R public $out/
+    '';
```

Comment:
```suggestion
    pname = "gitea-frontend";
    inherit (gitea) src version meta;

    npmDepsHash = "sha256-gXBBiDIIS0aW6qK37HcF0AuJOliblinznRVXoo6DV1s=";

    # use webpack directly instead of 'make frontend' as the packages are already installed
    buildPhase = ''
      BROWSERSLIST_IGNORE_OLD_DATA=true npx webpack
    '';

    installPhase = ''
      mkdir -p $out
      cp -R public $out/
    '';
```